apps/dashboard/config/initializers/activejobs.rb

```diff
@@ -0,0 +1,2 @@
+
+OODClusters = OodCore::Clusters.new(OodAppkit.clusters.select(&:allow?))
```

Comment:
We use a custom initializer https://github.com/OSC/osc-ood-config/blob/master/ondemand.osc.edu/apps/activejobs/initializers/ood_clusters_fix.rb alter this, though we may no longer be in need of this. I gather that is what you meant by backwards compatibility with custom initializers.

Reply:
We override it yes, and some other folks may as well. But this addition to dashboard's initializers is because activejobs also had this.

https://github.com/OSC/ondemand/blob/a0204bb9a9cbda12fd00d38c8eadfac73283bbb3/apps/activejobs/config/initializers/ood_appkit.rb